consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/jsonrpc/IbftJsonRpcMethodsFactory.java

```diff
@@ -34,10 +37,17 @@
     final Map<String, JsonRpcMethod> rpcMethods = new HashMap<>();
 
     if (jsonRpcApis.contains(IbftRpcApis.IBFT)) {
+      BlockchainQueries blockchainQueries =
+          new BlockchainQueries(context.getBlockchain(), context.getWorldStateArchive());
```

Comment:
nit: might be _slightly_ nicer to store context.X() things in local variables and use them from there ... but its 50/50.